packages/clay-select/src/index.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {SelectHTMLAttributes} from 'react';
+
+export interface ISelectOption {
+	label: string;
+	selected?: boolean;
+	value: string;
+}
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+	/**
+	 * Props to add to the outer most container.
+	 */
+	containerProps?: React.HTMLAttributes<HTMLDivElement>;
+
+	/**
+	 * Flag to indicate if the select is disabled or not.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Id attribute of the select.
+	 */
+	id?: string;
+
+	/**
+	 * Flag to display element `inline`.
+	 */
+	inline?: boolean;
+
+	/**
+	 * Label of the select.
+	 */
+	label: string;
+
+	/**
+	 * Flag to indicate if multiple options can be selected.
+	 */
+	multiple?: boolean;
+
+	/**
+	 * Name attribute of the select.
+	 */
+	name?: string;
+
+	/**
+	 * Options of the select.
+	 */
+	options: Array<ISelectOption>;
+}
+
+const ClaySelect: React.FunctionComponent<Props> = ({
+	className,
+	containerProps = {},
+	disabled,
+	id,
+	inline,
+	label,
+	multiple,
+	name,
+	onChange,
+	options,
+	...otherProps
+}) => {
+	return (
+		<div
+			{...containerProps}
+			className={classNames({
+				[`${containerProps.className}`]: containerProps.className,
+				'form-group': !inline,
+				'form-group-item': inline,
+			})}
+		>
+			{label && <label htmlFor={id}>{label}</label>}
+
+			<select
+				{...otherProps}
+				aria-label={label}
+				className={classNames('form-control', className)}
+				disabled={disabled}
+				id={id}
+				multiple={multiple}
+				name={name}
+				onChange={onChange}
+			>
+				{options.map((option, index) => (
+					<option
+						key={index}
+						selected={option.selected}
```

Comment:
Rather than having `option.selected`  how do you feel about `selectedValue` as a prop on `ClaySelect`? And then in this arg we would have `selected={option.value === selectedValue}`?

I did something like this with Radio, https://github.com/liferay/clay/blob/develop/packages/clay-radio-group/src/index.tsx#L110

I'd be curious too if React specifically recommends a paradigm for this.